frontend/azlinux/handler.go

```diff
@@ -21,10 +21,9 @@ const (
 type installFunc func(context.Context, gwclient.Client, dalec.SourceOpts) (llb.RunOption, error)
 
 type worker interface {
-	Base(sOpt dalec.SourceOpts, opts ...llb.ConstraintsOpt) (llb.State, error)
+	Base(sOpt dalec.SourceOpts, opts ...llb.ConstraintsOpt) llb.State
```

Comment:
Why remove the error and silently return `llb.Scratch()`?

Reply:
Good find, I don't know what was going on here...